Unexpected failure of selenium tests which are create workspaces from Dashboard

### Description
We have unexpected fail of selenium tests that create workspaces from **Dashboard**. The tests, sometimes, fail while waiting that **Project Explorer** visibility. We should clarify reason and fix it.

**Log:**
```
org.openqa.selenium.TimeoutException:  Expected condition failed: waiting for visibility of element located by By.id: gwt-debug-projectTree (tried for 240 second(s) with 500 MILLISECONDS interval) Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700' System info: host: 'slave6.codenvycorp.com', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.10.0-514.16.1.el7.x86_64', java.version: '1.8.0_102' Driver info: org.eclipse.che.selenium.core.SeleniumWebDriver 	at org.eclipse.che.selenium.dashboard.ImportProjectFromZipTest.importProjectFromZipTest(ImportProjectFromZipTest.java:83) Caused by: org.openqa.selenium.NoSuchElementException:  no such element: Unable to locate element: {"method":"id","selector":"gwt-debug-projectTree"}
```

**Report with reproduced problem:**
https://ci.codenvycorp.com/view/qa/job/che-integration-tests-multiuser-master-ocp/146/Selenium_tests_report/

--- a/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java
+++ b/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java
@@ -255,6 +255,8 @@ public class ProjectExplorer {
       if (seleniumWebDriverHelper.isVisible(By.id("ide-loader-progress-bar"))) {
         fail("Known issue https://github.com/eclipse/che/issues/8468", ex);
       }
+
+      throw ex;
     }
   }
 

--- a/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java
+++ b/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java
@@ -255,8 +255,6 @@ public class ProjectExplorer {
       if (seleniumWebDriverHelper.isVisible(By.id("ide-loader-progress-bar"))) {
         fail("Known issue https://github.com/eclipse/che/issues/8468", ex);
       }
-
-      fail("Known issue https://github.com/eclipse/che/issues/9100", ex);
     }
   }
 

